scale-out-relay/src/main/scala/com/normation/plugins/scaleoutrelay/ScaleOutRelayService.scala

```diff
@@ -59,6 +61,7 @@ class ScaleOutRelayService(
                     } else {
                       nodeInfos.succeed
                     }
+      _ <- clearCacheService.clearNodeConfigurationCache(storeEvent = true, actor)
```

Comment:
In fact you should trigger a deployment not delete the whole configurationCache.

We used to call deploy api that only starts a deployment:

```
      asyncDeploymentAgent ! AutomaticStartDeployment(modId, actor)
```

with 
```
  , asyncDeploymentAgent : AsyncDeploymentActor
```

you can put it in the yield section